subsys/bluetooth/host/smp.c

```diff
@@ -5802,12 +5802,12 @@ int bt_smp_start_security(struct bt_conn *conn)
 
 		/* pairing is in progress */
 		if (atomic_test_bit(smp->flags, SMP_FLAG_PAIRING)) {
-			return -EBUSY;
+			return -EINPROGRESS;
```

Comment:
not sure if we should change this just for sake of 'better' return code  (ie it may break existing application that is already checking against EBUSY)